Doc/whatsnew/3.14.rst

```diff
@@ -103,7 +103,9 @@ Optimizations
 Deprecated
 ==========
 
-
+* Deprecate ``Py_IS_NAN``, ``Py_IS_INFINITY`` and ``Py_IS_FINITE`` macroses,
```

Comment:
This is the section for Python deprecations, there's another further down for C API deprecations. Please move this there.

Reply:
Done.

BTW, shouldn't bedevere detect that this pr *has* a news entry?

Reply:
A news entry is a small file in https://github.com/python/cpython/tree/main/Misc/NEWS.d/next which is used to generate the changelog: https://docs.python.org/3.14/whatsnew/changelog.html#changelog

See step 8 at https://devguide.python.org/